test/unit/utils/nodeStarter.spec.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright © 2023 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+import { isLiskCoreV3Running } from '../../../src/utils/nodeStarter';
+
+describe('isLiskCoreV3Running', () => {
```

Comment:
The tests need to improvisation after the implementation is fixed.

Reply:
Applied.